airbyte-integrations/connectors/source-amplitude/source_amplitude/api.py

```diff
@@ -148,7 +151,12 @@ class Events(IncrementalAmplitudeStream):
     compare_date_template = "%Y-%m-%d %H:%M:%S.%f"
     primary_key = "uuid"
     state_checkpoint_interval = 1000
-    time_interval = {"days": 1}
+
+    @property
+    def time_interval(self) -> dict:
+        return {
+            self.event_time_interval.get('size_unit'): self.event_time_interval.get("size")
+        }
```

Comment:
For users which are upgrading to this version of the connector from a pervious version, we probably should provide a default value if there wasn't `config.event_time_interval` set before

Reply:
There is a default value for `size_unit` and `size` in the spec so which means default value for `event_time_interval` also.

Reply:
@evantahler can you update your review here !?

Reply:
Thank you for the explanation @mariamthiam! This PR still needs a version bump and a changelog entry, per the note above

Reply:
I think adding the default value in the config are helpful, but for users who are upgrading from the previous version of the connector, the config they will have from their previous version of the connector won't have any value for size_unit or size.  you can try this locally by removing the new object from your config file and trying a sync.    

I suggest you add some default behavior in the code here that matches the defaults in the connector spec

Reply:
I have added it 👍